### PR TITLE
Refactor PR format checking for easier extensions

### PR DESCRIPTION
**Pull request by xstefank**

### CODE REVIEW

**Comment by xstefank** [comment]
@Stanislav-Ko please review.

File: src/main/java/io/xstefank/wildlfy/bot/config/Format.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package io.xstefank.wildlfy.bot.config;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class Format {
+
+//    @ConfigProperty(name = "title-check")
```
Comment (skoshchi) [inline src/main/java/io/xstefank/wildlfy/bot/config/Format.java:7]:
```suggestion

```

Comment (skoshchi) [reply src/main/java/io/xstefank/wildlfy/bot/config/Format.java:7]: Delete a comment

**Review by skoshchi — APPROVED** [review]

File: src/main/java/io/xstefank/wildlfy/bot/config/Format.java (hunk 1 above)
Comment (xstefank) [reply src/main/java/io/xstefank/wildlfy/bot/config/Format.java:7]: We will fix this in next commit.

**Pull request merged**